python/pipeline/meso.py

```diff
@@ -918,9 +918,10 @@ def _make_tuples(self, key):
             dj.conn()
 
             ## Insert in CNMF, Segmentation and Fluorescence
-            Segmentation().insert1(key)
-            Segmentation.CNMF().insert1({**key, 'params': json.dumps(kwargs)})
-            Fluorescence().insert1(key)  # we also insert traces
+            # Segmentation().insert1(key)
+            # Segmentation.CNMF().insert1({**key, 'params': json.dumps(kwargs)})
+            self.insert1({**key, 'params': json.dumps(kwargs)})
```

Comment:
I would rather have Segmentation.insert1 here (even with allow_direct_insert) so it parallels the Manual insertions (i.e, CNMF). Either that or down there in the main, have self.insert1(key) for Manual too.